src/obfl-specification.html

```diff
@@ -1614,6 +1644,18 @@ <h4 id="L735" class="include">table-of-contents</h4>
           <dd>An id.</dd>
       </dl>
     </dd>
+  <dt>Variables</dt>
+    <dd><dl>
+        <dt>started-volume-number</dt>
+          <dd>The variable holding the number of the started volume. It can be
+            used in an expression, for instance, <code>&lt;evaluate
+            expression="$started-volume-number"/&gt;</code>.</dd>
+        <dt>started-page-number</dt>
+          <dd>The variable holding the number of the started page. It can be
+            used in an expression, for instance, <code>&lt;evaluate
+            expression="$started-page-number"/&gt;</code>.</dd>
+      </dl>
+    </dd>
```

Comment:
Not sure about this section. Where would you use these variables?

Reply:
It might be used within a `toc-entry`.
I have removed this section.

Reply:
Does it make sense to use these in a toc-entry? Maybe `started-volume-number`, although I can't think of a use case.

`started-page-number` however is something related to endnotes. I don't think it has a place here.